### PR TITLE
Add napari-allencell-annotator recipe

### PR DESCRIPTION
**Pull request by goanpeca**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- ~[ ] If static libraries are linked in, the license of the static library is packaged.~
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by goanpeca** [comment]
We are in the process of creating conda packages for [napari plugins](https://www.napari-hub.org/) and related dependencies. Once this recipe passes all the checks, every time a new release goes into PyPI, a PR will be created automatically, with usually very minor need for changes so the process of maintaining this moving forward is small. Also the `@conda-forge/napari` team will be available to help as needed.

You can read a bit more about conda-forge at https://conda-forge.org/#contribute

---

@jaimergp, @bbridge0200, @yrkim98 do you agree with being added as maintainers?

**Comment by goanpeca** [comment]
Haven't heard from the author in some time. The author can be added as maintainer in a subsequent PR.

**Comment by goanpeca** [comment]
This one is ready for review @conda-forge/staged-recipes :)

File: recipes/napari-allencell-annotator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "napari-allencell-annotator" %}
+{% set version = "0.0.5" %}
```
Comment (jaimergp) [inline recipes/napari-allencell-annotator/meta.yaml:2]:
```suggestion
{% set version = "0.0.7" %}
```

File: recipes/napari-allencell-annotator/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-allencell-annotator-{{ version }}.tar.gz
+  sha256: 1f3e3d42240c14dd98b594e6799c67247acab1beea5e497ea2a5e71c887c52d7
```
Comment (jaimergp) [inline recipes/napari-allencell-annotator/meta.yaml:10]:
```suggestion
  sha256: beccb3e9aeb1262bb5fc4e7be064cb935e415c7fa263fac7df42284cd5f2bac4
```

File: recipes/napari-allencell-annotator/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.8
```
Comment (jaimergp) [inline recipes/napari-allencell-annotator/meta.yaml:21]:
```suggestion
    - python >=3.7
```

File: recipes/napari-allencell-annotator/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  run:
+    - aicsimageio >=4.0.5
+    - aicssegmentation >=0.4.1
+    - magicgui >=0.2.9
+    - napari
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - pyqt
+    - python >=3.8
```
Comment (jaimergp) [inline recipes/napari-allencell-annotator/meta.yaml:30]:
```suggestion
    - python >=3.7
```

File: recipes/napari-allencell-annotator/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    - pytest-runner
+    - python >=3.8
+  run:
+    - aicsimageio >=4.0.5
+    - aicssegmentation >=0.4.1
+    - magicgui >=0.2.9
+    - napari
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - pyqt
```
Comment (jaimergp) [inline recipes/napari-allencell-annotator/meta.yaml:29]:
```suggestion
    - psygnal
```

File: recipes/napari-allencell-annotator/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.8
+  run:
+    - aicsimageio >=4.0.5
+    - aicssegmentation >=0.4.1
+    - magicgui >=0.2.9
+    - napari
```
Comment (jaimergp) [inline recipes/napari-allencell-annotator/meta.yaml:26]:
```suggestion
    - napari >=0.4.9
```

**Pull request merged**